client/components/notice/index.jsx

```diff
@@ -65,9 +71,17 @@ export const Notice = React.createClass( {
 			}
 
 			content = [ this.props.children ];
-			content.unshift( <span key="notice_text" className="notice__text">{ text }</span> );
+			content.unshift(
+				<span className="notice__content">
+					<span key="notice_text" className="notice__text">{ text }</span>
```

Comment:
The `key` prop will need to be moved to the parent component. I expect there's a warning in the console with this change.